.prettierrc

```diff
@@ -9,7 +9,12 @@
   ],
   "overrides": [
     {
-      "files": ["data/*.json", "schemas/**/*.json", "test/fixtures/*.json"],
+      "files": [
+        "data/*.json",
+        "schemas/**/*.json",
+        "test/fixtures/*.json",
```

Comment:
is fixtures still necessary?

Reply:
Yeah, there's some stuff in there that is actually test fixtures (sample incentive relationships, etc.)